admin-dev/themes/new-theme/tests/cldr/number-symbol.spec.js

```diff
@@ -1,3 +1,27 @@
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
```

Comment:
I dont know if we should put license headers on JS tests files 🤔 we do it for php files, but the goal of license headers is to be shipped in the Zip Archive, right ? so these tests are fine

By the way @eternoendless we should consequently ignore these files in the Release Builder, right ?

Reply:
> I dont know if we should put license headers on JS tests files

People can still pick up our files from Github so I think we should put the header on ALL files.

> we should consequently ignore these files in the Release Builder, right ?

Yes we should